vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -12,129 +12,199 @@ import "RedBoxHandler.idl";
 
 namespace Microsoft.ReactNative {
 
+  DOC_STRING(
+    "A JavaScript engine type that can be set for a React instance in @ReactInstanceSettings.JSIEngineOverride")
   // Keep in sync with enum in IReactInstance.h
-  enum JSIEngine {
+  enum JSIEngine
+  {
     Chakra = 0,
     Hermes = 1,
     V8 = 2
   };
 
   [webhosthidden]
   [default_interface]
-  runtimeclass InstanceCreatedEventArgs {
+  DOC_STRING("The arguments for the @ReactInstanceSettings.InstanceCreated event.")
+  runtimeclass InstanceCreatedEventArgs
+  {
+    DOC_STRING("Gets @IReactContext for the React instance that was just created.")
     IReactContext Context { get; };
   }
 
   [webhosthidden]
   [default_interface]
-  runtimeclass InstanceLoadedEventArgs {
+  DOC_STRING("The arguments for the @ReactInstanceSettings.InstanceLoaded event.")
+  runtimeclass InstanceLoadedEventArgs
+  {
+    DOC_STRING("Gets @IReactContext for the React instance that finished loading the bundle.")
     IReactContext Context { get; };
+
+    DOC_STRING("Gets `true` if JavaScript bundle failed to load.")
     Boolean Failed { get; };
   }
 
   [webhosthidden]
-  [default_interface] runtimeclass InstanceDestroyedEventArgs {
+  [default_interface]
+  DOC_STRING("The arguments for the @ReactInstanceSettings.InstanceDestroyed event.")
+  runtimeclass InstanceDestroyedEventArgs
+  {
+    DOC_STRING("Gets @IReactContext for the React instance that just destroyed.")
```

Comment:
```suggestion
    DOC_STRING("Returns the @IReactContext for the React instance that just destroyed.")
```

Reply:
I think we use the verb 'returns' for functions and the 'gets' for properties, right?

Reply:
I keep the verb "Gets"